test/widgets/action_sheet_test.dart

```diff
@@ -490,4 +491,32 @@ void main() {
       check(await Clipboard.getData('text/plain')).isNull();
     });
   });
+
+  group('CopyMessageLinkButton', () {
+    setUp(() async {
+      TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.setMockMethodCallHandler(
```

Comment:
nit: I'd rather not add the leading whitespace
```suggestion
TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.setMockMethodCallHandler(
```